xbmc/cores/RetroPlayer/savestates/SavestateFlatBuffer.cpp

```diff
@@ -135,19 +136,34 @@ void CSavestateFlatBuffer::SetLabel(const std::string& label)
   m_labelOffset.reset(new StringOffset{m_builder->CreateString(label)});
 }
 
+std::string CSavestateFlatBuffer::Caption() const
+{
+  std::string caption;
+
+  if (m_savestate != nullptr && m_savestate->caption())
+    caption = m_savestate->caption()->c_str();
+
+  return caption;
+}
+
+void CSavestateFlatBuffer::SetCaption(const std::string& caption)
+{
+  m_captionOffset.reset(new StringOffset{m_builder->CreateString(caption)});
+}
+
 CDateTime CSavestateFlatBuffer::Created() const
 {
   CDateTime created;
 
   if (m_savestate != nullptr && m_savestate->created())
-    created.SetFromRFC1123DateTime(m_savestate->created()->c_str());
+    created.SetFromW3CDateTime(m_savestate->created()->c_str());
```

Comment:
```suggestion
    createdUTC.SetFromW3CDateTime(m_savestate->created()->c_str(), false);
```

Using `UTC` in the variable name indicates to the reader that the time is UTC.

Even though the second parameter is false be default, being explicit indicates to the reader that the default value is intentional. Lots of times I've removed the default from parameters and some uses left me scratching my head what the original author meant.